### PR TITLE
Support multiple async function calls.

### PR DESCRIPTION
**Pull request by platfowner**
Summary:
- Support multiple function calls
- Add more test cases

#188

### CODE REVIEW

**Review by minsulee2 — APPROVED** [review]
LGTM

File: db/functions.js [hunk 1]
```diff
@@ -94,15 +97,23 @@ class Functions {
                 `  ==> Triggering an event for function '${functionEntry.function_id}' ` +
                 `of '${functionEntry.event_listener}' ` +
                 `with transaction: ${JSON.stringify(transaction, null, 2)}`)
-            return axios.post(functionEntry.event_listener, {
+            promises.push(axios.post(functionEntry.event_listener, {
               transaction,
               function: functionEntry
-            });
+            }));
```
Comment (liayoo) [inline db/functions.js:103]: How about a catch here?

Comment (platfowner) [reply db/functions.js:112]: Done. Thanks!

**Comment by platfowner** [comment]
Comments addressed. PTAL @liayoo

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Pull request merged**